__tests__/components/AppLayout/SidebarItems/SettingsNavigationItem.comprehensive.test.tsx

```
@@ -88,141 +116,155 @@ const mockUseSettingsNormal = () => ({
 });
 
 // Mock setup for failed update
-const mockUpdateSettingsFail = jest.fn().mockResolvedValue({ 
-  ok: false, 
-  statusText: "Server Error" 
+const mockUpdateSettingsFail = jest.fn().mockResolvedValue({
+  ok: false,
+  statusText: 'Server Error',
 });
 
 const mockUpdateSettingsSuccess = jest.fn().mockResolvedValue({ ok: true });
```

ESLint

[error] 'mockUpdateSettingsSuccess' is assigned a value but never used.

(@typescript-eslint/no-unused-vars)